#! /bin/sh /usr/share/dpatch/dpatch-run
## 201_configure.diff, author unknown
##
## DP: The configure script for slrn searches the $PATH for inews. We don't
## DP: want to install a news server at build time so we hardcode the path.

@DPATCH@
diff --git a/autoconf/configure.ac b/autoconf/configure.ac
--- a/autoconf/configure.ac
+++ b/autoconf/configure.ac
@@ -97,7 +97,9 @@ CF_COMPILE_OPTION(inews,
     [SLRN_HAS_INEWS_SUPPORT], 0)
 if test "x$enable_inews" = xyes ; then
   dnl# find inews
-  AC_PATH_PROG(INEWS, inews, no, $PATH:/usr/local/bin:/usr/bin)
+  dnl# always on for debian
+  dnl# AC_PATH_PROG(INEWS, inews, no, $PATH:/usr/local/bin:/usr/bin)
+  ac_cv_path_INEWS=/usr/bin/inews
   if test "x$ac_cv_path_INEWS" != xno; then
       dnl# AC_DEFINE_UNQUOTED(SLRN_INEWS_COMMAND, "$ac_cv_path_INEWS -S -h")
       AC_SUBST(SLRN_INEWS_COMMAND, "$ac_cv_path_INEWS -S -h")
